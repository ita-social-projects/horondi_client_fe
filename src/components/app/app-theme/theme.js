const profilePageLabel = {
  normal: {
    backgroundColor: 'rgba(196, 196, 196, 0.2)'
  },
  hover: {
    backgroundColor: 'rgba(196, 196, 196, 0.5)'
  }
};

export default {
  light: {
    textColor: '#000000',
    backgroundColor: '#FFFFFF',
    catalogPageFilterCell: {
      hover: {
        backgroundColor: '#e4e0e0'
      }
    },
    button: {
      normal: {
        backgroundColor: '#020202',
        color: '#FFFFFF'
      },
      hover: {
        backgroundColor: '#000000',
        color: '#FFFFFF'
      },
      disabled: {
        backgroundColor: '#999999',
        color: '#C2C2C2'
      },
      border: {
        borderColor: '#020202'
      }
    },
    card: {
      backgroundColor: '#F5F5F5',
      border: '1px solid #D3D3D3',
      childrenBackgroundColor: '#FFFFFF',
      selectedButton: {
        backgroundColor: '#000000',
        color: '#FFFFFF'
      }
    },
    cart: {
      borderColor: 'rgba(91, 91, 91, 0.2)'
    },
    contactForm: {
      background: '#EFEFEF'
    },
    orderHistoryHeading: {
      background: '#DEDEDE'
    },
    profilePageLabel,
    carouselItem: {
      normal: {
        backgroundColor: '#FFFFFF'
      },
      hover: {
        border: '1px solid #5b5b5b'
      }
    }
  },
  dark: {
    textColor: '#FFFFFF',
    backgroundColor: '#242424',
    catalogPageFilterCell: {
      hover: {
        backgroundColor: '#7c7b7b'
      }
    },
    button: {
      normal: {
        backgroundColor: '#020202',
        color: '#FFFFFF'
      },
      hover: {
        backgroundColor: '#FFFFFF',
        color: '#000000'
      },
      disabled: {
        backgroundColor: '#868585',
        color: '#C2C2C2'
      },
      border: {
        borderColor: '#FEFEFE'
      }
    },
    card: {
      backgroundColor: '#454545',
      border: '1px solid #282828',
      childrenBackgroundColor: '#353535',
      selectedButton: {
        backgroundColor: '#FFFFFF',
        color: '#000000'
      }
    },
    cart: {
      borderColor: '#5B5B5B'
    },
    contactForm: {
      background: '#232323'
    },
    orderHistoryHeading: {
      background: '#2F2F2F '
    },
    profilePageLabel,
    carouselItem: {
      normal: {
        backgroundColor: '#333131'
      },
      hover: {
        border: '1px solid #5B5B5B'
      }
    }
  }
};
