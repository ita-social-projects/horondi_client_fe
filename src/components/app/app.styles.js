import { makeStyles } from '@material-ui/core/styles';

export const useAppStyles = makeStyles((theme) => ({
  app: {
    overflowY: 'scroll'
  },
  center: {
    width: '3rem',
    margin: '22rem auto',
    '@media (max-width: 1400px)': {
      margin: '13rem auto'
    }
  },
  body: {
    backgroundColor: '#fff'
  },
  mainBar: {
    minHeight: '100vh',
    display: 'flex',
    flexDirection: 'column'
  },
  rootApp: {
    width: '95vw',
    margin: '0 auto',
    color: theme.palette.textColor
  },
  containerApp: {
    maxWidth: 1100,
    margin: '0 auto'
  }
}));
