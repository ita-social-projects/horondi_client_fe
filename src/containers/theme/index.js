import ThemeComponent from './theme-component';

export default ThemeComponent;
