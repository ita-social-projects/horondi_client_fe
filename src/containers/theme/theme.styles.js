import { makeStyles } from '@material-ui/core/styles';

export const useStyles = makeStyles((theme) => ({
  root: () => ({
    display: 'flex',
    marginLeft: '12px',
    marginRight: '12px',
    alignItems: 'center',
    '& .MuiTypography-root': {
      width: '40px',
      textAlign: 'center'
    },
    '& .MuiSwitch-root': {
      width: '32px',
      height: '14px',
      padding: '0px',
      margin: '0 5px',
      display: 'flex'
    },
    '& .MuiSwitch-switchBase': {
      padding: 2,
      color: '#808080',
      '&.Mui-checked': {
        transform: 'translateX(17px)',
        color: '#242424',
        '& + .MuiSwitch-track': {
          opacity: 1,
          backgroundColor: '#808080',
          borderColor: theme.palette.primary.main
        }
      }
    },
    '& .MuiSwitch-thumb': {
      width: 10,
      height: 10,
      boxShadow: 'none'
    },
    '& .MuiSwitch-track': {
      border: `1px solid ${theme.palette.grey[500]}`,
      borderRadius: '12px',
      opacity: 1,
      backgroundColor: theme.palette.common.white
    },
    '& .Mui-checked': {}
  })
}));
