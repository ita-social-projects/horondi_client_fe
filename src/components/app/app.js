import React, { useEffect } from 'react';
import { useDispatch, useSelector } from 'react-redux';
import { ThemeProvider } from '@material-ui/styles';
import { CssBaseline } from '@material-ui/core';
import Routes from '../../routes';
import Chat from '../../containers/chat';
import SearchBarList from '../../containers/search-bar-list';
import { theme } from './app-theme/app.theme';
import { LIGHT_THEME } from '../../configs';
import { useStyles } from './app.styles';
import { getFromLocalStorage } from '../../services/local-storage.service';
import { setThemeMode } from '../../redux/theme/theme.actions';
import { getCategories } from '../../redux/categories/categories.actions';
import { preserveUser } from '../../redux/user/user.actions';
import {
  getAllFilters,
  setCountPerPage,
  getFiltredProducts
} from '../../redux/products/products.actions';
import { getContacts } from '../../redux/contacts/contacts.actions';
import { selectLightModeAndLocation } from '../../redux/selectors/multiple.selectors';

const App = () => {
  const { lightMode, location } = useSelector(selectLightModeAndLocation);
  const dispatch = useDispatch();
  const styles = useStyles({ isHome: location === '/' });

  let localStorageThemeMode = getFromLocalStorage('theme');
  const themeMode = localStorageThemeMode === LIGHT_THEME;
  if (!localStorageThemeMode) {
    localStorageThemeMode = LIGHT_THEME;
  }
  const themeValue = theme(localStorageThemeMode);
  const productsCount = getFromLocalStorage('countPerPage');

  useEffect(() => {
    dispatch(preserveUser());
    dispatch(getCategories());
    dispatch(getContacts());
    dispatch(getAllFilters());
    dispatch(getFiltredProducts({}));
  }, []);

  useEffect(() => {
    dispatch(setCountPerPage(productsCount));
  }, [dispatch, productsCount]);

  useEffect(() => {
    dispatch(setThemeMode(themeMode));
  }, [lightMode, dispatch, themeMode]);

  return (
    <div className={styles.mainBar}>
      <ThemeProvider theme={themeValue}>
        <CssBaseline />
        <Routes />
        <Chat />
        <SearchBarList />
      </ThemeProvider>
    </div>
  );
};

export default App;
