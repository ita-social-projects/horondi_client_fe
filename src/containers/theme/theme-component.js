import React, { useContext } from 'react';
import { useTranslation } from 'react-i18next';
import { Switch, Typography } from '@material-ui/core';
import { DARK_THEME, LIGHT_THEME } from '../../configs';
import ThemeContext from '../../context/theme-context';
import { getFromLocalStorage, setToLocalStorage } from '../../services/local-storage.service';
import { useStyles } from './theme.styles';

const ThemeComponent = () => {
  const { t } = useTranslation();
  const [lightMode, setLightMode] = useContext(ThemeContext);
  const theme = getFromLocalStorage('theme');
  const styles = useStyles();

  const handleChangeTheme = (e) => {
    e.target.checked = !e.target.checked;
    setLightMode(!lightMode);
    setToLocalStorage('theme', !lightMode ? LIGHT_THEME : DARK_THEME);
  };

  return (
    <div className={styles.root}>
      <Typography>{t('header.lightTheme')}</Typography>
      <Switch checked={theme === 'dark'} onClick={handleChangeTheme} data-testid='switch-theme' />
      <Typography>{t('header.darkTheme')}</Typography>
    </div>
  );
};

export default ThemeComponent;
