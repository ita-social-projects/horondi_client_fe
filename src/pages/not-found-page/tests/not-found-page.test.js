import React from 'react';
import { useSelector, useDispatch } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import { mount } from 'enzyme';
import { ThemeProvider } from '@material-ui/styles';
import { theme } from '../../../components/app/app-theme/app.theme';
import NotFoundPage from '../not-found-page';

const mockDispatch = jest.fn();
const mockUseHistory = jest.fn();

jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useHistory: () => ({
    goBack: mockUseHistory
  })
}));

jest.mock('react-redux');

describe('', () => {
  let wrapper;

  beforeEach(() => {
    const themeValue = theme('light');

    useDispatch.mockImplementation(() => mockDispatch);
    useSelector.mockReturnValue({
      language: '0',
      isLightTheme: true
    });

    wrapper = mount(
      <BrowserRouter>
        <ThemeProvider theme={themeValue}>
          <NotFoundPage />
        </ThemeProvider>
      </BrowserRouter>
    );
  });

  afterEach(() => {
    wrapper.unmount();
    useDispatch.mockClear();
    useSelector.mockClear();
  });

  it('Should render 404 page', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('Should click back-page button', () => {
    const btn = wrapper.find('button');
    btn.simulate('click');

    expect(mockUseHistory).toHaveBeenCalledTimes(1);
  });

  it('Should work with dark theme', () => {
    const themeValue = theme('dark');

    useSelector.mockReturnValue({
      language: '0',
      isLightTheme: true
    });

    const Language = { language: 1 };
    const Theme = { lightMode: false };

    useSelector.mockImplementation((callback) => callback({ Language, Theme }));

    wrapper = mount(
      <BrowserRouter>
        <ThemeProvider theme={themeValue}>
          <NotFoundPage />
        </ThemeProvider>
      </BrowserRouter>
    );

    useSelector.mockClear();
  });
});
