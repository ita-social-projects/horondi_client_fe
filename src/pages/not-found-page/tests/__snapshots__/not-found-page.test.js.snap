// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render 404 page 1`] = `
<BrowserRouter>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <ThemeProvider
      theme={
        Object {
          "breakpoints": Object {
            "between": [Function],
            "down": [Function],
            "keys": Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ],
            "only": [Function],
            "up": [Function],
            "values": Object {
              "lg": 1280,
              "md": 960,
              "sm": 600,
              "xl": 1920,
              "xs": 0,
            },
            "width": [Function],
          },
          "direction": "ltr",
          "mixins": Object {
            "gutters": [Function],
            "toolbar": Object {
              "@media (min-width:0px) and (orientation: landscape)": Object {
                "minHeight": 48,
              },
              "@media (min-width:600px)": Object {
                "minHeight": 64,
              },
              "minHeight": 56,
            },
          },
          "overrides": Object {},
          "palette": Object {
            "action": Object {
              "activatedOpacity": 0.12,
              "active": "rgba(0, 0, 0, 0.54)",
              "disabled": "rgba(0, 0, 0, 0.26)",
              "disabledBackground": "rgba(0, 0, 0, 0.12)",
              "disabledOpacity": 0.38,
              "focus": "rgba(0, 0, 0, 0.12)",
              "focusOpacity": 0.12,
              "hover": "rgba(0, 0, 0, 0.04)",
              "hoverOpacity": 0.04,
              "selected": "rgba(0, 0, 0, 0.08)",
              "selectedOpacity": 0.08,
            },
            "augmentColor": [Function],
            "background": Object {
              "default": "#fafafa",
              "paper": "#fff",
            },
            "backgroundColor": "#FFFFFF",
            "black": "#000000",
            "button": Object {
              "disabled": Object {
                "backgroundColor": "#999999",
                "color": "#C2C2C2",
              },
              "hover": Object {
                "backgroundColor": "#000000",
                "color": "#FFFFFF",
              },
              "normal": Object {
                "backgroundColor": "#020202",
                "color": "#FFFFFF",
              },
            },
            "card": Object {
              "backgroundColor": "#F5F5F5",
              "border": "1px solid #D3D3D3",
              "childrenBackgroundColor": "#FFFFFF",
              "selectedButton": Object {
                "backgroundColor": "#000000",
                "color": "#FFFFFF",
              },
            },
            "common": Object {
              "black": "#000",
              "white": "#fff",
            },
            "contrastThreshold": 3,
            "divider": "rgba(0, 0, 0, 0.12)",
            "error": Object {
              "contrastText": "#FFFFFF",
              "dark": "#b71c1c",
              "light": "#ef5350",
              "main": "#e53935",
            },
            "getContrastText": [Function],
            "grey": Object {
              "100": "#f5f5f5",
              "200": "#eeeeee",
              "300": "#e0e0e0",
              "400": "#bdbdbd",
              "50": "#fafafa",
              "500": "#9e9e9e",
              "600": "#757575",
              "700": "#616161",
              "800": "#424242",
              "900": "#212121",
              "A100": "#d5d5d5",
              "A200": "#aaaaaa",
              "A400": "#303030",
              "A700": "#616161",
            },
            "icon": "#546e7a",
            "info": Object {
              "contrastText": "#FFFFFF",
              "dark": "#0d47a1",
              "light": "#42a5f5",
              "main": "#1e88e5",
            },
            "primary": Object {
              "contrastText": "#FFFFFF",
              "dark": "#212121",
              "light": "#212121",
              "main": "#212121",
            },
            "secondary": Object {
              "contrastText": "#FFFFFF",
              "dark": "#0d47a1",
              "light": "#2979ff",
              "main": "#2979ff",
            },
            "success": Object {
              "contrastText": "#FFFFFF",
              "dark": "#1b5e20",
              "light": "#66bb6a",
              "main": "#43a047",
            },
            "text": Object {
              "disabled": "rgba(0, 0, 0, 0.38)",
              "hint": "rgba(0, 0, 0, 0.38)",
              "primary": "rgba(0, 0, 0, 0.87)",
              "secondary": "rgba(0, 0, 0, 0.54)",
            },
            "textColor": "#000000",
            "tonalOffset": 0.2,
            "type": "light",
            "warning": Object {
              "contrastText": "#FFFFFF",
              "dark": "#e65100",
              "light": "#ffa726",
              "main": "#fb8c00",
            },
            "white": "#FFFFFF",
          },
          "props": Object {},
          "shadows": Array [
            "none",
            "0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12)",
            "0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12)",
            "0px 3px 3px -2px rgba(0,0,0,0.2),0px 3px 4px 0px rgba(0,0,0,0.14),0px 1px 8px 0px rgba(0,0,0,0.12)",
            "0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12)",
            "0px 3px 5px -1px rgba(0,0,0,0.2),0px 5px 8px 0px rgba(0,0,0,0.14),0px 1px 14px 0px rgba(0,0,0,0.12)",
            "0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12)",
            "0px 4px 5px -2px rgba(0,0,0,0.2),0px 7px 10px 1px rgba(0,0,0,0.14),0px 2px 16px 1px rgba(0,0,0,0.12)",
            "0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12)",
            "0px 5px 6px -3px rgba(0,0,0,0.2),0px 9px 12px 1px rgba(0,0,0,0.14),0px 3px 16px 2px rgba(0,0,0,0.12)",
            "0px 6px 6px -3px rgba(0,0,0,0.2),0px 10px 14px 1px rgba(0,0,0,0.14),0px 4px 18px 3px rgba(0,0,0,0.12)",
            "0px 6px 7px -4px rgba(0,0,0,0.2),0px 11px 15px 1px rgba(0,0,0,0.14),0px 4px 20px 3px rgba(0,0,0,0.12)",
            "0px 7px 8px -4px rgba(0,0,0,0.2),0px 12px 17px 2px rgba(0,0,0,0.14),0px 5px 22px 4px rgba(0,0,0,0.12)",
            "0px 7px 8px -4px rgba(0,0,0,0.2),0px 13px 19px 2px rgba(0,0,0,0.14),0px 5px 24px 4px rgba(0,0,0,0.12)",
            "0px 7px 9px -4px rgba(0,0,0,0.2),0px 14px 21px 2px rgba(0,0,0,0.14),0px 5px 26px 4px rgba(0,0,0,0.12)",
            "0px 8px 9px -5px rgba(0,0,0,0.2),0px 15px 22px 2px rgba(0,0,0,0.14),0px 6px 28px 5px rgba(0,0,0,0.12)",
            "0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12)",
            "0px 8px 11px -5px rgba(0,0,0,0.2),0px 17px 26px 2px rgba(0,0,0,0.14),0px 6px 32px 5px rgba(0,0,0,0.12)",
            "0px 9px 11px -5px rgba(0,0,0,0.2),0px 18px 28px 2px rgba(0,0,0,0.14),0px 7px 34px 6px rgba(0,0,0,0.12)",
            "0px 9px 12px -6px rgba(0,0,0,0.2),0px 19px 29px 2px rgba(0,0,0,0.14),0px 7px 36px 6px rgba(0,0,0,0.12)",
            "0px 10px 13px -6px rgba(0,0,0,0.2),0px 20px 31px 3px rgba(0,0,0,0.14),0px 8px 38px 7px rgba(0,0,0,0.12)",
            "0px 10px 13px -6px rgba(0,0,0,0.2),0px 21px 33px 3px rgba(0,0,0,0.14),0px 8px 40px 7px rgba(0,0,0,0.12)",
            "0px 10px 14px -6px rgba(0,0,0,0.2),0px 22px 35px 3px rgba(0,0,0,0.14),0px 8px 42px 7px rgba(0,0,0,0.12)",
            "0px 11px 14px -7px rgba(0,0,0,0.2),0px 23px 36px 3px rgba(0,0,0,0.14),0px 9px 44px 8px rgba(0,0,0,0.12)",
            "0px 11px 15px -7px rgba(0,0,0,0.2),0px 24px 38px 3px rgba(0,0,0,0.14),0px 9px 46px 8px rgba(0,0,0,0.12)",
          ],
          "shape": Object {
            "borderRadius": 4,
          },
          "spacing": [Function],
          "transitions": Object {
            "create": [Function],
            "duration": Object {
              "complex": 375,
              "enteringScreen": 225,
              "leavingScreen": 195,
              "short": 250,
              "shorter": 200,
              "shortest": 150,
              "standard": 300,
            },
            "easing": Object {
              "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
              "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
              "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
              "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
            },
            "getAutoHeightDuration": [Function],
          },
          "typography": Object {
            "body1": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "1rem",
              "fontWeight": 400,
              "lineHeight": 1.5,
            },
            "body2": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "0.875rem",
              "fontWeight": 400,
              "lineHeight": 1.43,
            },
            "button": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "16px",
              "fontWeight": 500,
              "lineHeight": 1.75,
              "textTransform": "uppercase",
            },
            "caption": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "11px",
              "fontWeight": 400,
              "letterSpacing": "0.33px",
              "lineHeight": "13px",
            },
            "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
            "fontSize": 14,
            "fontWeightBold": 700,
            "fontWeightLight": 300,
            "fontWeightMedium": 500,
            "fontWeightRegular": 400,
            "h1": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "35px",
              "fontWeight": 500,
              "letterSpacing": "-0.24px",
              "lineHeight": "40px",
            },
            "h2": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "29px",
              "fontWeight": 500,
              "letterSpacing": "-0.24px",
              "lineHeight": "32px",
            },
            "h3": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "24px",
              "fontWeight": 500,
              "letterSpacing": "-0.06px",
              "lineHeight": "28px",
            },
            "h4": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "20px",
              "fontWeight": 500,
              "letterSpacing": "-0.06px",
              "lineHeight": "24px",
            },
            "h5": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "16px",
              "fontWeight": 500,
              "letterSpacing": "-0.05px",
              "lineHeight": "20px",
            },
            "h6": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "14px",
              "fontWeight": 500,
              "letterSpacing": "-0.05px",
              "lineHeight": "20px",
            },
            "htmlFontSize": 16,
            "overline": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "11px",
              "fontWeight": 500,
              "letterSpacing": "0.33px",
              "lineHeight": "13px",
              "textTransform": "uppercase",
            },
            "pxToRem": [Function],
            "round": [Function],
            "subtitle1": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "16px",
              "fontWeight": 400,
              "letterSpacing": "-0.05px",
              "lineHeight": "25px",
            },
            "subtitle2": Object {
              "fontFamily": "Montserrat,-apple-system,Arial,sans-serif",
              "fontSize": "14px",
              "fontWeight": 400,
              "letterSpacing": "-0.05px",
              "lineHeight": "21px",
            },
          },
          "zIndex": Object {
            "appBar": 1100,
            "drawer": 1200,
            "mobileStepper": 1000,
            "modal": 1300,
            "snackbar": 1400,
            "speedDial": 1050,
            "tooltip": 1500,
          },
          Symbol(mui.nested): false,
        }
      }
    >
      <NotFoundPage>
        <div
          className="makeStyles-wrapper-1"
        >
          <div>
            <div
              className="makeStyles-info-6"
            >
              <h2>
                4
                <span
                  className="makeStyles-imageContainer-4"
                >
                  <img
                    alt="notFoundPage.message"
                    className="makeStyles-image-5"
                    src="./images/not-found-page/bag.png"
                  />
                </span>
                4
              </h2>
              <h3>
                notFoundPage.message
              </h3>
              <p>
                notFoundPage.toHome
                 
                <Link
                  className="makeStyles-link-2"
                  data-cy="home"
                  to="/"
                >
                  <LinkAnchor
                    className="makeStyles-link-2"
                    data-cy="home"
                    href="/"
                    navigate={[Function]}
                  >
                    <a
                      className="makeStyles-link-2"
                      data-cy="home"
                      href="/"
                      onClick={[Function]}
                    >
                      notFoundPage.linkHome
                    </a>
                  </LinkAnchor>
                </Link>
              </p>
              <WithStyles(ForwardRef(Button))
                className="makeStyles-button-3"
                data-cy="back"
                onClick={[Function]}
                variant="contained"
              >
                <ForwardRef(Button)
                  className="makeStyles-button-3"
                  classes={
                    Object {
                      "colorInherit": "MuiButton-colorInherit",
                      "contained": "MuiButton-contained",
                      "containedPrimary": "MuiButton-containedPrimary",
                      "containedSecondary": "MuiButton-containedSecondary",
                      "containedSizeLarge": "MuiButton-containedSizeLarge",
                      "containedSizeSmall": "MuiButton-containedSizeSmall",
                      "disableElevation": "MuiButton-disableElevation",
                      "disabled": "Mui-disabled",
                      "endIcon": "MuiButton-endIcon",
                      "focusVisible": "Mui-focusVisible",
                      "fullWidth": "MuiButton-fullWidth",
                      "iconSizeLarge": "MuiButton-iconSizeLarge",
                      "iconSizeMedium": "MuiButton-iconSizeMedium",
                      "iconSizeSmall": "MuiButton-iconSizeSmall",
                      "label": "MuiButton-label",
                      "outlined": "MuiButton-outlined",
                      "outlinedPrimary": "MuiButton-outlinedPrimary",
                      "outlinedSecondary": "MuiButton-outlinedSecondary",
                      "outlinedSizeLarge": "MuiButton-outlinedSizeLarge",
                      "outlinedSizeSmall": "MuiButton-outlinedSizeSmall",
                      "root": "MuiButton-root",
                      "sizeLarge": "MuiButton-sizeLarge",
                      "sizeSmall": "MuiButton-sizeSmall",
                      "startIcon": "MuiButton-startIcon",
                      "text": "MuiButton-text",
                      "textPrimary": "MuiButton-textPrimary",
                      "textSecondary": "MuiButton-textSecondary",
                      "textSizeLarge": "MuiButton-textSizeLarge",
                      "textSizeSmall": "MuiButton-textSizeSmall",
                    }
                  }
                  data-cy="back"
                  onClick={[Function]}
                  variant="contained"
                >
                  <WithStyles(ForwardRef(ButtonBase))
                    className="MuiButton-root MuiButton-contained makeStyles-button-3"
                    component="button"
                    data-cy="back"
                    disabled={false}
                    focusRipple={true}
                    focusVisibleClassName="Mui-focusVisible"
                    onClick={[Function]}
                    type="button"
                  >
                    <ForwardRef(ButtonBase)
                      className="MuiButton-root MuiButton-contained makeStyles-button-3"
                      classes={
                        Object {
                          "disabled": "Mui-disabled",
                          "focusVisible": "Mui-focusVisible",
                          "root": "MuiButtonBase-root",
                        }
                      }
                      component="button"
                      data-cy="back"
                      disabled={false}
                      focusRipple={true}
                      focusVisibleClassName="Mui-focusVisible"
                      onClick={[Function]}
                      type="button"
                    >
                      <button
                        className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-3"
                        data-cy="back"
                        disabled={false}
                        onBlur={[Function]}
                        onClick={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        tabIndex={0}
                        type="button"
                      >
                        <span
                          className="MuiButton-label"
                        >
                          notFoundPage.linkBack
                        </span>
                        <WithStyles(memo)
                          center={false}
                        >
                          <ForwardRef(TouchRipple)
                            center={false}
                            classes={
                              Object {
                                "child": "MuiTouchRipple-child",
                                "childLeaving": "MuiTouchRipple-childLeaving",
                                "childPulsate": "MuiTouchRipple-childPulsate",
                                "ripple": "MuiTouchRipple-ripple",
                                "ripplePulsate": "MuiTouchRipple-ripplePulsate",
                                "rippleVisible": "MuiTouchRipple-rippleVisible",
                                "root": "MuiTouchRipple-root",
                              }
                            }
                          >
                            <span
                              className="MuiTouchRipple-root"
                            >
                              <TransitionGroup
                                childFactory={[Function]}
                                component={null}
                                exit={true}
                              />
                            </span>
                          </ForwardRef(TouchRipple)>
                        </WithStyles(memo)>
                      </button>
                    </ForwardRef(ButtonBase)>
                  </WithStyles(ForwardRef(ButtonBase))>
                </ForwardRef(Button)>
              </WithStyles(ForwardRef(Button))>
            </div>
          </div>
        </div>
      </NotFoundPage>
    </ThemeProvider>
  </Router>
</BrowserRouter>
`;
